packages/devextreme/testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -173,7 +173,7 @@ QUnit.module('Initialization', baseModuleConfig, () => {
         assert.equal(filterRowWrapper.getTextEditorInput(1).attr('aria-describedby'), headersWrapper.getHeaderItem(0, 4).attr('id'));
 
         assert.equal(dataGridWrapper.getElement().find('.dx-datagrid').attr('role'), 'group', 'group role');
-        assert.equal(dataGridWrapper.getElement().find('.dx-datagrid').attr('aria-label'), 'Data grid with 2 rows and 4 columns', 'aria-label of the datagrid container');
+        assert.equal(dataGridWrapper.getElement().find('.dx-datagrid').attr('aria-label'), 'Data grid with 2 rows and 4 columns. ', 'aria-label of the datagrid container');
```

Comment:
I noticed there is no similar test for TreeList

Reply:
I have added it to the existing testcase I created